content/docs/configuration.md

```diff
@@ -277,6 +277,15 @@ notifications:
 
 You can enable the commenting by setting `successful_build` to `true`.
 
+#### issue_repository
+
+Use this key to be notified about errors of the downstream jobs (Koji build and Bodhi update).
+This repository can be any GitHub/GitLab/Pagure repository
+where issues are enabled and Packit has an identity on that instance.
+(Let us know if you need some other instance to be supported.)
```

Comment:
```suggestion
This repository can be any GitHub/GitLab/Pagure instance
where issues are enabled and Packit has an identity.
(Let us know if you need some other instances to be supported.)
```

Reply:
I might have mixed those two config options together, but here, we are really talking about repositories. As I am looking at that, I'll make it more clear and mention that it needs to be a URL.

Reply:
Can you please take another look if it is clear now?